### PR TITLE
prepare for 5.0.1 release

### PR DESCRIPTION
**Pull request by cwjohnston**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [x] Update Changelog following the conventions laid out [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

#### Purpose

Release changes to sys-filesystem dependency, bonsai asset builds, readme as 5.0.1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -210,7 +212,8 @@ https://mathias-kettner.de/checkmk_filesystems.html
 ### Added
 - initial release
 
-[Unreleased]: https://github.com/sensu-plugins/sensu-plugins-disk-checks/compare/5.0.0...HEAD
+[Unreleased]: https://github.com/sensu-plugins/sensu-plugins-disk-checks/compare/5.0.1...HEAD
+[5.0.0]: https://github.com/sensu-plugins/sensu-plugins-disk-checks/compare/5.0.0...5.0.1
```
Comment (cwjohnston) [inline CHANGELOG.md]:
```suggestion
[5.0.1]: https://github.com/sensu-plugins/sensu-plugins-disk-checks/compare/5.0.0...5.0.1
```

**Comment by cwjohnston** [comment]
2019-12-11 at 10 43 AM

**Comment by jspaleta** [comment]
:+1:

**Comment by majormoses** [comment]
@cwjohnston we have all been there...

**Pull request merged**